### PR TITLE
BootstrapNAS: Export best subnet to onnx

### PR DESCRIPTION
**Pull request by jpablomch**
### Changes

Add calls in examples to export the resulting best subnet to ONNX

### Reason for changes

Example of exporting to ONNX requested by user. 

### Related tickets

N/A

### Tests

Test for exporting to ONNX are already included in test suite.

### CODE REVIEW

**Comment by jpablomch** [comment]
@ljaljushkin Please take a look. Thanks!

**Review by ljaljushkin — APPROVED** [review]

**Comment by ljaljushkin** [comment]
This PR can't be merged without passing all pre-commit checks.

There's issue with code style, please refer to [pylint check log](https://ci-adas-icv.iotg.sclab.intel.com/job/NNCF/job/precommit/job/pylint/120/).

Also, one e2e test for efficient (nas.test_sanity_sample.test_e2e_supernet_training[efficient_net_b1_cifar10_nas.json_classification_nas_mock_32x32]) is failing with error: `RuntimeError: ONNX export failed: Couldn't export Python operator SwishImplementation`.

Seems like there's an option to use export-friendly version of this model: https://github.com/lukemelas/EfficientNet-PyTorch/issues/91#issuecomment-542994572
Please use that version in the test.

**Comment by jpablomch** [comment]
run build

**Comment by jpablomch** [comment]
run build

**Comment by jpablomch** [comment]
run build

File: examples/torch/common/models/classification/efficientnet.py [hunk 1]
```diff
@@ -23,5 +23,8 @@
 
 def efficient_net(pretrained=True, num_classes=1000, **kwargs):
     if pretrained:
-        return EfficientNet.from_pretrained(num_classes=num_classes, **kwargs)
-    return EfficientNet.from_name(num_classes=num_classes, **kwargs)
+        model = EfficientNet.from_pretrained(num_classes=num_classes, **kwargs)
+    else:
+        model = EfficientNet.from_name(num_classes=num_classes, **kwargs)
+    model.set_swish(memory_efficient=False)
```
Comment (ljaljushkin) [inline examples/torch/common/models/classification/efficientnet.py:29]: It breaks backward compatibility with previous interface. That's why tests for searching blocks fail. 
Propose to call `set_swish` after model creation when it's needed, and not do it by default on creation.

Comment (jpablomch) [reply examples/torch/common/models/classification/efficientnet.py]: Thanks! I reverted this change and I am calling `set_swish` after model creation.

**Comment by jpablomch** [comment]
run build

**Pull request merged**